src/app/collection/pages/collection-ncyte/curators/curator-card/curator-card.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { User } from '@entity';
+import { UserService } from 'app/core/user.service';
+
+@Component({
+  selector: 'clark-curator-card',
+  templateUrl: './curator-card.component.html',
+  styleUrls: ['./curator-card.component.scss']
+})
+export class CuratorCardComponent implements OnInit {
+
+  @Input() curator: any;
+  profileImg: string | undefined;
+
+  constructor(private userService: UserService) { }
+
+  ngOnInit(): void {
+    console.log(this.curator);
+    this.profileImg = this.userService.getGravatarImage(this.curator.email, 100);
```

Comment:
What are we going to do if a curator does not use gravatar?

Reply:
Gravatar has an autogenerated image it uses if you don't have one.  I don't have one and it makes one for me.